
From: "Randy.Dunlap" <rddunlap@osdl.org>

'make buildcheck' indicates that these functions should not be in an __exit
section, so undo that.  Yes, they can be called from .text.

Signed-off-by: Randy Dunlap <rddunlap@osdl.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/sound/oss/awe_wave.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN sound/oss/awe_wave.c~awe_wave-oss-too-much-__exit sound/oss/awe_wave.c
--- 25/sound/oss/awe_wave.c~awe_wave-oss-too-much-__exit	2004-08-04 20:31:25.612763928 -0700
+++ 25-akpm/sound/oss/awe_wave.c	2004-08-04 20:31:25.619762864 -0700
@@ -4130,7 +4130,7 @@ static void __init attach_mixer(void)
 	}
 }
 
-static void __exit unload_mixer(void)
+static void unload_mixer(void)
 {
 	if (my_mixerdev >= 0)
 		sound_unload_mixerdev(my_mixerdev);
@@ -4968,7 +4968,7 @@ static void __init attach_midiemu(void)
 		midi_devs[my_mididev] = &awe_midi_operations;
 }
 
-static void __exit unload_midiemu(void)
+static void unload_midiemu(void)
 {
 	if (my_mididev >= 0)
 		sound_unload_mididev(my_mididev);
_
